src/vos/vos_ts.h

```diff
@@ -587,6 +587,15 @@ vos_ts_evict(uint32_t *idx, uint32_t type)
 	lrua_evict(ts_table->tt_type_info[type].ti_array, idx);
 }
 
+static inline bool
+vos_ts_lookup_entry(uint32_t *idx, uint32_t type, struct vos_ts_entry **entryp)
+{
+	struct vos_ts_table	*ts_table = vos_ts_table_get();
+	struct vos_ts_info	*info = &ts_table->tt_type_info[type];
+
+	return lrua_lookup(info->ti_array, idx, entryp);
```

Comment:
The one concern I have here is that lrua_lookup makes the entry the MRU which probably isn't a side effect we want from aggregation.   I think it would be better to implement an lrua_peek() and maybe modify lrua_lookup_idx to take an extra argument to skip the call to lrua_move_to_mru

Reply:
Ok, let me check.